src/index.ts

```diff
@@ -8,8 +8,18 @@ export { Watcher } from './Watcher';
 export * from './interfaces';
 export * from './LanguageServer';
 export * from './XmlScope';
-export * from './lexer';
```

Comment:
All the imports from these removed lines were expanded out to their individual file exports. This ensures that our "public" api hasn't changed. One caveat, if plugin authors doing things like `import { createVisitor } from 'brighterscript/astUtils' then this is technically a breaking change. However, since we were already exporting those directly from `src/index.ts`, I'd argue they were not using our library properly. 

Thoughts?